README.md

```diff
@@ -223,6 +223,7 @@ The following table lists the configurable parameters of the Harbor chart and th
 | `core.secretName`                                                           | Fill the name of a kubernetes secret if you want to use your own TLS certificate and private key for token encryption/decryption. The secret must contain keys named: `tls.crt` - the certificate and `tls.key` - the private key. The default key pair will be used if it isn't set                                                                                                                                                                                                           |                                                                                     |
 | `core.xsrfKey`                                                              | The XSRF key. Will be generated automatically if it isn't specified                                                                                                                                                                                                                                                                                                                                                                                                                            |                                                                                     |
 | `core.priorityClassName`                                                    | The priority class to run the pod as                                                                                                                                                                                                                                                                                                                                                                                                                                                           |                                                                                     |
+| `core.enableMigrateHelmHook`                                                | Run the migration job via helm hook                                                                                                                                                                                                                                                                                                                                                                                                                                                            | `false`                                                                             |
```

Comment:
I don't think the option should be put under `core`

Reply:
updated